### DISCUSSION TITLE
Add a feature to create folders or areas again

### DISCUSSION THREAD

**Discussion by Suzie97** [alainm23/planify#652] [General]
I used to use folders as the main projects and projects as my sub projects, but since I updated to 2.6 using flatpak, my folders have dissappeared. I know that I can sort my projects into sub projects but that is going to take a lot of time. 

Is there any way I can convert my current projects into sub projects?

Or can you provide the option to create folders again?

**Comment by rzelnik** [comment]
I was confused too. Currently the folders do not exist, but you can create super-projects instead of folders and move the existing projects into them. It's a bit tricky, because usual drag & drop works only when you open the parent project. There is an easier way: right-click on existing project, move > choose the parent project.

**Reply by Suzie97** [reply]
I was a bit lazy to do that because I have a lot of projects. But I guess that is the only way.

Todoist doesn't have folder support, so porjects and subprojects is a better choice as they can be synced.
